### PR TITLE
#169 - Add more unit tests

### PR DESCRIPTION
**Pull request by hirako2000**
This PR is related to issue #169
Not everything is done. I raise two different issues to address the problem with the tests.

### CODE REVIEW

**Comment by hirako2000** [comment]
I added `coverage` to .gitignore so that we can easily run coverage when running tests locally.

File: test/components.spec.js [hunk 1]
```diff
@@ -65,3 +64,26 @@ test('Group', () => {
   const group = new WHS.Group(sphere, box, new Mesh());
   group.addTo(app);
 });
+
+describe('Line', () => {
+  test('throws if no curve', () => {
+    expect(() => {
+      new WHS.Line();
+    }).toThrow();
+  });
```
Comment (sasha240100) [inline test/components.spec.js]: `CSF`: new line before/after code block.

File: test/modules.spec.js [hunk 2]
```diff
@@ -62,8 +62,11 @@ test('Applying controls modules to app', () => {
   app.start();
 });
 
-test('DynamicGeometryModule', () => {
+describe('DynamicGeometryModule', () => {
   modules.dym = new WHS.mesh.DynamicGeometryModule();
```
Comment (sasha240100) [inline test/modules.spec.js]: `CSF`: new line before/after code block.

File: test/components.spec.js [hunk 3]
```diff
@@ -65,3 +64,26 @@ test('Group', () => {
...
+  test('has points passed in', () => {
+    // TODO assert, I suspect this component is broken
+  });
+});
+
+describe('Model', () => {
+  test('can construct', () => {
+    const model = new WHS.Model();
```
Comment (sasha240100) [inline test/components.spec.js]: `CSF`: Unused variable declaration `model`.

File: test/components.spec.js [hunk 4]
```diff
@@ -65,3 +64,26 @@ test('Group', () => {
...
+  });
+});
+
+describe('Text', () => {
+  test('can construct', () => {
+    const text = new WHS.Text();
```
Comment (sasha240100) [inline test/components.spec.js]: `CSF`: Unused variable declaration `text`.

**Pull request merged**